src/lib/schemaParser.test.ts

```diff
@@ -131,11 +133,13 @@ describe('schemaParser getSchema', () => {
     it('finds known SomeBytes32Mapping:<bytes32>', () => {
       const bytes32Value =
         '1111222233334444555566667777888899990000aaaabbbbccccddddeeeeffff';
-      const name = `SomeBytes32Mapping:${bytes32Value}`;
+      const name = `SomeBytes32Mapping:<bytes32>`;
+      const dynamicName = `SomeBytes32Mapping:${bytes32Value}`;
       const key = `0x0cfc51aec37c55a4d0b10000${bytes32Value.slice(0, 42)}`;
 
-      const extraSchema: ERC725JSONSchema = {
+      const extraSchema: DynamicNameSchema = {
         name,
+        dynamicName,
```

Comment:
Hmmm, this should really be dynamicKeyParts: string | string[]. You're welcome to have dynamicName, but when passing it to encodeKey it will expect dynamicKeyParts, no?